src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* max-width: 1200px;
+  min-width: 1024px; */
+  padding: 0 40px 0 50px;
+  margin: 0 auto;
+  background-color: #fff;
+  box-shadow: 0 2px 4px #e2e7ea;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
+}
+
+.logo {
+  position: relative;
+  bottom: 3px;
+}
+
+.nav_link {
+  
+  display: inline-block;
+  font-weight: 500;
+  font-size: 12px;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+  line-height: 60px;
+}
+
+.nav_list:last-child {
```

```suggestion
.nav_item:last-child {
```
Це для nav_item потрібно задавати